lib/debugletapi.js

```diff
@@ -83,7 +83,7 @@ DebugletApi.prototype.init = function(uid, logger, callback) {
         logger.warn('Malformed source-context.json file.');
         // But we keep on going.
       }
-      callback(null);
+      callback(null, project);
```

Comment:
Now the project name is passed to the callback in the error case as well as the non-error case.